subsys/bluetooth/controller/ll_sw/ull_sync_iso.c

```diff
@@ -384,6 +425,30 @@ void ull_sync_iso_setup(struct ll_sync_iso_set *sync_iso,
 	lll->payload_count |= (uint64_t)bi->payload_count_framing[3] << 24;
 	lll->payload_count |= (uint64_t)bi->payload_count_framing[4] << 32;
 
+	if (lll->enc && (bi_size == PDU_BIG_INFO_ENCRYPTED_SIZE)) {
+		const uint8_t BIG3[4]  = {0x33, 0x47, 0x49, 0x42};
+		struct ccm *ccm_rx;
+		uint8_t gsk[16];
+		int err;
+
+		/* Copy the GIV in BIGInfo */
+		(void)memcpy(lll->giv, bi->giv, sizeof(lll->giv));
+
+		/* Calculate GSK */
+		err = bt_crypto_h8(sync_iso->gltk, bi->gskd, BIG3, gsk);
+		LL_ASSERT(!err);
+
+		/* Prepare the CCM parameters */
+		ccm_rx = &lll->ccm_rx;
+		ccm_rx->direction = 1U;
```

Comment:
The `direction` values 0 and 1, which has special meaning (central/peripheral, right?) should IMO be `define`d